help_macros.json

```diff
@@ -304,25 +304,25 @@
     "flags": [
       "incomplete"
     ],
-    "teamplay-restricted": true
+    "teamplay-restricted": false
   },
   "pointatloc": {
     "flags": [
       "incomplete"
     ],
-    "teamplay-restricted": true
+    "teamplay-restricted": false
   },
   "pointloc": {
     "flags": [
       "incomplete"
     ],
-    "teamplay-restricted": true
+    "teamplay-restricted": false
   },
   "powerups": {
     "flags": [
       "incomplete"
     ],
-    "teamplay-restricted": true
+    "teamplay-restricted": false
```

Comment:
Hmmm... this is definitely a bug, but what ruleset did you have enabled when you generated the help file? :)